deploy/docs/V1_Migration_doc.md

```diff
@@ -0,0 +1,166 @@
+# Kubernetes Collection 1.0.0 - Breaking Changes
+
+<!-- TOC -->
+
+- [Helm Users](#helm-users) 
+  - [Changes](#changes) 
+  - [How to upgrade](#how-to-upgrade) 
+    - [1. Upgrade to helm chart version v0.17.1](#1-upgrade-to-helm-chart-version-v0.17.1)
+    - [2: Run upgrade script](#2-run-upgrade-script)
+- [Non-Helm Users](#non-helm-users) 
+  - [Changes](#breaking-changes) 
+  - [How to upgrade](#how-to-upgrade-for-non-helm-users)
+    - [1. Move environment variable config changes to Fluentd pipeline](#1-move-environment-variable-config-changes-to-fluentd-pipeline)
+    - [2. Modify filter based on kubernetes version](#2-modify-filter-based-on-kubernetes-version))
+    - [3: Apply new changes](#3-apply-new-changes)
+
+
+<!-- /TOC -->
+
+Based on the feedback from our users, we will be introducing several changes to the Sumo Logic Kubernetes Collection solution. Here we detail the changes for both Helm and Non-Helm users, as well as the exact steps for migration.
+## Helm Users
+### Changes
+
+- Falco installation disabled by Default
+
+- Bumped the helm Falco chart version which included a fix to disable the bitcoin miner rule by default
```

Comment:
@frankreno Can you actually demarcate what version you bumped the falco dependency to?

Reply:
falco version `1.1.6` . Updated in the doc.